Test/task1.c

```diff
@@ -0,0 +1,84 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+unsigned int collection[1000];
+
+unsigned int fibonacci(int number);
+bool fibonacciTest();
+
+int main()
+{
+    if(!fibonacciTest())
+    {
+        return 1;
+    }
+    
+    int sum = 0;
+    
+    for(int i = 0; i < 100; i++)
```

Comment:
```suggestion
    for (int i = 0; i < 100; i++)
```
Соответственно, и ниже